_book/05-toolkit-reference/04-toolkit-options.md

```diff
@@ -25,12 +25,14 @@ see-also:
   <label><input type="radio" name="lang" checked>Command-line parameters</label>
 </div>
 <div class="hidden-print radio-inline">
-  <label><input type="radio" name="lang">JSON keys</label>
+  <label><input type="radio" name="lang">JSON objects</label>
```

Comment:
```suggestion
  <label><input type="radio" name="lang">JavaScript options</label>
```

Reply:
I suspect most people will care about where they're used, not how they're passed. The fact that it's JSON is secondary to it being used to pass the options to the toolkit.